packages/cli/src/index.ts

```diff
@@ -652,6 +660,11 @@ yargs
           description: 'Project directory.',
           default: '.',
         },
+        'lint-config': {
```

Comment:
I noticed that `-p` is not working for this command, can we add it?

Reply:
What would you expect `-p` to do here? We have both `--plan` and `--port`, so we didn't add shorteners. Or were you thinking of another option that should be controlled with `-p` ?

Reply:
`-p` is a common alias for `--port`. This is how it works with realm CLI.

Reply:
Added the alias.